package.json

```diff
@@ -13,14 +13,16 @@
         "build": "npm run clear-dist && npm run compile",
         "make-dist": "npm run build && npm run bundle && node dist/transpiled/scripts/make-dist-v2.js build-all --project-root './'",
         "build-artifacts": "npm-run-all build make-dist",
+        "sync-version": "node dist/transpiled/scripts/sync-version.js",
```

Comment:
could use ts-node instead for this i guess?

Reply:
yes. done.